src/lib/components/PricingPlan.svelte

```
@@ -0,0 +1,121 @@
+    title: string;
+    description: string;
+    price: string;
+    priceSubtext: string;
+    features: string[];
+    buttonText: string;
+    buttonVariant: "default" | "outline" | "secondary" | "destructive" | "ghost";
+    buttonAction: () => void;
+    isPopular: boolean;
+    isPrimary: boolean;
+    customBadgeText?: string | null;
+    customBadgeIcon?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)